samples/samples/backup_sample_test.py

```diff
@@ -113,3 +119,13 @@ def test_cancel_backup(capsys):
         "Backup deleted." in out
     )
     assert cancel_success or cancel_failure
+
+
+@RetryErrors(exception=DeadlineExceeded, max_tries=2)
+def test_create_database_with_retention_period(capsys):
+    database_id = unique_database_id()
```

Comment:
nit: prefer to declare these globally (see other IDs at the top)
This applies to `retention_period` as well